lib/cmd/fh3/keys/ssh/add.js

```diff
@@ -0,0 +1,52 @@
+/* globals i18n */
+var common = require('../../../../common.js');
+var fhreq = require("../../../../utils/request");
+var fs = require('fs');
+var fhc = require("../../../../fhc");
+
+module.exports = {
+  'desc' : i18n._('Add a SSH Key Management'),
```

Comment:
Why `SSH Key Management`? Isn't this just adding a single SSH Key? Maybe better name it `Add a new SSH key`?